packages/core/src/entity/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum PostMessageMethods {
```

Comment:
Not necessary to be done in this PR but just wanna call out that we should avoid enums as long as they are not necessary. Our [TS standards doc](https://contentful.atlassian.net/wiki/spaces/ENG/pages/4189094335/TypeScript+Standards#Enums) recommends using union types or object literals.